source/controllers/SectionController.ts

```diff
@@ -1,16 +1,146 @@
+import { isAdmin, isLoggedOn } from '../middleware/AdminCheckMiddleware.js';
 import { Section } from '../models/Section.js';
-import { Controller } from './BaseController.js';
 import { type User } from '../models/User.js';
+import { Controller } from './BaseController.js';
+
 import { headerConstants } from './config.js';
 
 const section = new Controller('section');
 
 section.read = async (req, res) => {
 	const sections = await Section.findAll();
-	if (res.locals.user !== null && res.locals.user !== undefined) { // for header Login&sign up or username& sign out
+	if (isLoggedOn(res.locals.user)) { // for header Login&sign up or username& sign out
```

Comment:
I'd like to see `isLoggedOn` used everywhere instead of `(res.locals.user !== null && res.locals.user !== undefined)`